test/testcases/conftest.py

```
@@ -279,13 +292,13 @@ def set_tenant_info(auth):
     set_default_embedding_response = requests.patch(
         url=url, headers=authorization, json={"model_provider": "Builtin", "model_instance": "Local", "model_type": "embedding", "model_name": "BAAI/bge-small-en-v1.5"}
     )
-    embd_res = set_default_embedding_response.json()
+    embd_res = _response_json_or_warning(set_default_embedding_response, "set default embedding LLM")
     if embd_res.get("code") != 0:
         print(f"WARNING: failed to set default embedding LLM via {url}: {embd_res.get('message')!r}. Continuing.")
     # set rerank model
     set_default_rerank_response = requests.patch(
```

Ruff (0.15.20)

[error] Probable use of `requests` call without timeout

(S113)